projects/plugins/boost/app/assets/src/js/features/page-cache/meta/meta.tsx

```diff
@@ -39,29 +89,121 @@ const Meta = () => {
 			</div>
 			{ isExpanded && (
 				<div className={ styles.body }>
-					<div className={ styles.section }>
-						<div className={ styles.title }>{ __( 'Exceptions', 'jetpack-boost' ) }</div>
-						<p>{ __( 'URLs of pages and posts that will never be cached:', 'jetpack-boost' ) }</p>
-						<textarea rows={ 3 }></textarea>
-						<p className={ styles.description }>
-							{ __(
-								'Use (*) to address multiple URLs under a given path. Be sure each URL path is in its own line. See an example or learn more.',
-								'jetpack-boost'
-							) }
-						</p>
-						<Button>{ __( 'Save', 'jetpack-boost' ) }</Button>
-					</div>
-					<div className={ styles.section }>
-						<div className={ styles.title }>{ __( 'Logging', 'jetpack-boost' ) }</div>
-						<label htmlFor="cache-logging">
-							<input type="checkbox" id="cache-logging" />{ ' ' }
-							{ __( 'Activate logging to track all your cache events.', 'jetpack-boost' ) }
-						</label>
-					</div>
+					{ settings && (
+						<>
+							<Exceptions
+								exceptions={ settings.exceptions.join( '\n' ) }
+								setExceptions={ setExceptions }
+								showErrorNotice={ mutation.isError }
+							/>
+							<div className={ styles.section }>
+								<div className={ styles.title }>{ __( 'Logging', 'jetpack-boost' ) }</div>
+								<label htmlFor="cache-logging">
+									<input
+										type="checkbox"
+										id="cache-logging"
+										checked={ settings.logging }
+										onChange={ event => setLogging( event.target.checked ) }
+									/>{ ' ' }
+									{ __( 'Activate logging to track all your cache events.', 'jetpack-boost' ) }
+									{ settings.logging && (
+										<>
+											{ ' ' }
+											<Link to="/cache-debug-log">{ __( 'See Logs', 'jetpack-boost' ) }</Link>
+										</>
+									) }
+								</label>
+							</div>
+						</>
+					) }
 				</div>
 			) }
 		</div>
 	);
 };
 
 export default Meta;
+
+type ExceptionsProps = {
+	exceptions: string;
+	setExceptions: ( newValue: string ) => void;
+	showErrorNotice: boolean;
+};
+
+const Exceptions = ( { exceptions, setExceptions, showErrorNotice = false }: ExceptionsProps ) => {
+	const [ inputValue, setInputValue ] = useState( exceptions );
+	const [ showNotice, setShowNotice ] = useState( showErrorNotice );
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	const [ inputInvalid, setInputInvalid ] = useState( false ); // @todo - implement this
+
+	// @todo - add proper link.
+	const exclusionsLink = 'TBD';
```

Comment:
Use `https://jetpack.com` for now so that the link renders corretly.

Reply:
Thanks. Updated.